README.md

```diff
@@ -1,36 +1,44 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## About
 
-## Getting Started
+College Predictor provides the list of colleges given :
```

Comment:
`provides a list of colleges with opening and closing ranks given:`

Reply:
updated